source/utilities.tex

```diff
@@ -14554,7 +14554,7 @@
     function() noexcept;
     function(nullptr_t) noexcept;
     function(const function&);
-    function(function&&);
+    function(function&&) noexcept;;
```

Comment:
Extra semicolon

Reply:
thanks for catching this cut&paste error!